orb.yml

```diff
@@ -29,7 +29,7 @@ commands:
       - run:
           name: downloading buildevents executables
           command: |
-            curl -q -L -o /tmp/be/bin-linux/buildevents https://github.com/honeycombio/buildevents/releases/download/v0.4.5/buildevents-linux-amd64
+            curl -q -L -o /tmp/be/bin-linux/buildevents https://522-172153261-gh.circle-artifacts.com/0/go/src/github.com/honeycombio/buildevents/artifacts/buildevents-linux-amd64
```

Comment:
is this to catch the new buildevents bits?  should we hold off until v0.4.6 and keep the urls clean? :)

Reply:
yes, this was to publish a test version of the orb that I could use in another repo (https://github.com/maplebed/circlecitesting) to verify the fix. Now that buildevents is versioned this change is gone and it uses v0.4.6 as you suggest.